Cargo.toml

```
@@ -46,8 +47,8 @@ indicatif = "0.17.3"
 glob = { version = "0.3.1", optional = true }
 
 [features]
-default = ["std", "fast-alloc", "cpu-par-matmul"]
-nightly = []
+default = ["std", "fast-alloc", "cpu-par-matmul", "test-f16", "nightly"]
```

```suggestion
default = ["std", "fast-alloc", "cpu-par-matmul", "f16", "nightly"]
```
Testing with f32 should be the default for now.